dm/dm.go

```diff
@@ -71,13 +72,15 @@ var usage = func() {
 	fmt.Fprintln(os.Stderr, "Flags:")
 	flag.PrintDefaults()
 	fmt.Fprintln(os.Stderr)
-	os.Exit(1)
+	fmt.Fprintln(os.Stderr, "--stdin requires a tar archive of input files and a top level file name.")
+	fmt.Fprintln(os.Stderr)
+	panic("\n")
 }
 
 func getGitRegistry() *registry.GithubRegistry {
 	s := strings.Split(*template_registry, "/")
 	if len(s) < 2 {
-		log.Fatalf("invalid template registry: %s", *template_registry)
+		panic(fmt.Sprintf("invalid template registry: %s", *template_registry))
```

Comment:
What's wrong with log.Fatal? I think panic dumps a stacktrace and seems not that user friendly to dump the stack?